Add error code for operations that require implicit transaction (#385)
neo4j/docs-status-codes: modules/ROOT/pages/errors/gql-errors/25N17.adoc

:page-role: new-2025.10

= 25N17

== Status description
error: The attempted operation requires an implicit transaction.

== Scenario

This error occurs when the running a query using the CALL { ... } IN TRANSACTIONS construct in an explicit transaction.

== Possible solution

> Make sure to use the driver API for creating an implicit transaction, or if using Browser/Query use the :auto command.

Reviewer: ```suggestion
Make sure you use the driver API to create an implicit transaction, or, if using Browser/Query, use the `:auto` command.
```
